### PR TITLE
[Merged by Bors] - feat: Add `Zspan.fundamentalDomain_ae_parallelepiped`

### PR DESCRIPTION
**Pull request by xroblot**
Prove that the fundamental domain of a `ℤ`-lattice is almost equal to the corresponding parallelepiped. This is useful because many results in measure theory use the parallelepiped associated to a basis.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Module/Zlattice.lean [hunk 1]
```diff
@@ -365,6 +370,30 @@ theorem volume_fundamentalDomain [Fintype ι] [DecidableEq ι] (b : Basis ι ℝ
     mul_one, ← Matrix.det_transpose]
   rfl
 
+theorem fundamentalDomain_ae_parallelepiped [Fintype ι] [MeasurableSpace E] (μ : Measure E)
+    [BorelSpace E] [Measure.IsAddHaarMeasure μ] :
+    fundamentalDomain b =ᶠ[Measure.ae μ] parallelepiped b := by
```
Comment (sgouezel) [inline Mathlib/Algebra/Module/Zlattice.lean:375]:
```suggestion
    fundamentalDomain b =ᵐ[μ] parallelepiped b := by
```
It's the same thing, but more readable.

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

**Comment by Ruben-VandeVelde** [comment]
`Error: ./././Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean:62:4: error: unknown identifier 'SMulHomClass.map_smul'` was presumably from here. Please fix and re-submit

**Pull request closed without merging**